docs/changes.rst

```diff
@@ -29,6 +29,8 @@ Released: not yet
 * Fix issue where the instance shrub --fullpath option was not displaying the
   paths. (see issue #1180)
 
+* Fixed new formatting issues raised by flake8 5.0.
```

Comment:
The change log entry does not match at all with the changes in the PR.

Also, I cannot relate the PR title with the changes in the PR.

Can you please rebase this PR to master, and make its PR title, commit message and change log consistent with the actual changes?

Reply:
1. Entry made in change log
2. The changes in the PR are
   * one line change to pywbemcli.py
   * Add tests
 3. Rebased to master